amf-cli/shared/src/test/resources/validations/reports/model/draft-2019-unsupported-facets-warning.report

```diff
@@ -1,18 +1,10 @@
 ModelId: file://amf-cli/shared/src/test/resources/validations/raml/draft-2019/draft-2019-unsupported-facets-warning.raml
 Profile: RAML 1.0
 Conforms: true
-Number of results: 7
+Number of results: 4
 
 Level: Warning
 
-- Constraint: http://a.ml/vocabularies/amf/validation#untranslatable-draft-2019-fields
```

Comment:
Why this ??

Reply:
these validations seemed to be duplicated, saying the same thing twice.